ddp-workspace/projects/ddp-pancan/src/app/components/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="wrapper">
     <app-header></app-header>
     <router-outlet></router-outlet>
-    <app-footer></app-footer>
+    <app-footer *ngIf="!isSplashPage"></app-footer>
```

Comment:
I'd double check  that colorectal page has different footer design (in the same time header is the same)

Reply:
That is it. 
From the [ticket](https://broadinstitute.atlassian.net/browse/DDP-6640) description:
```
3. The top header in the page is the same header as the rest of the Pan Can page/none of the links in the header point to anything within the colorectal splash page. Only back to the already established Pan Can FAQ/Participation etc.
4. Note that the footer IS unique to colorectal with its own email, etc.
```